internal/gps/vcs_source_test.go

```diff
@@ -524,6 +524,62 @@ func testHgSourceInteractions(t *testing.T) {
 	<-donech
 }
 
+func Test_gitSource_listVersions_noHEAD(t *testing.T) {
```

Comment:
I think we decided to not do any more snake case function names, idiomatic go. The ones that you see are old code, we need to change them. So, let's rename this to `TestGitSourceListVersionsNoHEAD`.